third_party/2and3/click/types.pyi

```diff
@@ -270,7 +270,9 @@ class UUIDParameterType(ParamType):
         ...
 
 
-def convert_type(ty: Any, default: Optional[Any] = ...) -> ParamType:
+_ConvertibleType = Union[type, ParamType, PyTuple[type, ...], Callable[[str], Any], Callable[[Optional[str]], Any]]
```

Comment:
Isn't `Callable[[str], Any]` redundant with `Callable[[Optional[str]], Any]`?

Reply:
It doesn't seem to be.  

```
from typing import Union, Callable, Optional, Any

With = Union[None, Callable[[str], Any], Callable[[Optional[str]], Any]]
Without = Union[None, Callable[[Optional[str]], Any]]

def cb1(a: str) -> None:...
def cb2(a: Optional[str]) -> None: ...

def fn_with(cb: With) -> None: ...
def fn_without(cb: Without) -> None: ...

fn_with(cb1)
fn_with(cb2)

fn_without(cb1)   # Argument 1 to "fn_without" has incompatible type "Callable[[str], None]"; expected "Optional[Callable[[Optional[str]], Any]]"
fn_without(cb2)
```